### PR TITLE
[inductor] Add scaled_dot_product_attention to fallback kernels

### PR DESCRIPTION
**Pull request by bertmaher**
Summary:
We don't have decomps/lowerings for SDPA (and probably won't for a
while) so don't warn.

Test Plan: code inspection

Differential Revision: D42878203

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

### CODE REVIEW

**Review by desertfire — APPROVED** [review]

File: torch/_inductor/lowering.py [hunk 1]
```diff
@@ -1194,6 +1194,8 @@ def apply_constraint(arg, fx_arg):
 make_fallback(aten._fused_moving_avg_obs_fq_helper_functional)
 make_fallback(aten.grid_sampler_2d_backward, require_dense)
 make_fallback(aten.randperm)
+make_fallback(aten._scaled_dot_product_efficient_attention.default)
```
Comment (drisspg) [inline torch/_inductor/lowering.py:1197]: could you also add the flash_attention variants as well?
After this PR: https://github.com/pytorch/pytorch/pull/92917 they will start to show up

**Comment by eellison** [comment]
Thanks !

**Review by drisspg — APPROVED** [review]

**Pull request closed without merging**